frontend/src/pages/LoginPage.jsx

```diff
@@ -0,0 +1,86 @@
+import { Box, TextField, Typography, useTheme, Button } from '@mui/material';
+import { useState } from 'react';
+import logo from '../assets/igym-logo-text.png';
+import AddIcon from '@mui/icons-material/Add';
+
+export default function LoginPage() {
+    const [isCreatingAccount, setIsCreatingAccount] = useState(false);
+    const theme = useTheme();
+    const textFieldSx = {
+
+        '& .MuiOutlinedInput-root': {
+            '& fieldset': {
+                borderColor: theme.palette.secondary.main,
+            },
+            '&:hover fieldset': {
+                borderColor: theme.palette.secondary.main,
+            },
+            '&.Mui-focused fieldset': {
+                borderColor: theme.palette.secondary.main,
+            },
+        },
+
+        '& .MuiInputLabel-root': {
+            color: theme.palette.secondary.main,
+        },
+        '& label.Mui-focused': {
+            color: theme.palette.secondary.main,
+        },
+
+        input: {
+            color: theme.palette.secondary.main,
+        },
+    }
+
+    return (
+        <Box
+            display="flex"
+            flexDirection="column"
+            alignItems="center"
+            justifyContent="center"
+            minHeight="90vh"
```

Comment:
```suggestion

```
This shorthand prop is forcing an unnecessary right-sidebar scrolling at the page. Anyway I don't know if was your intention to make the colors of the scroll bar almost invisible compared with the background color, I think is a good way but I'm not sure about this color. Considerer try another color.